### PR TITLE
Migrate from using react-icons

### PR DESCRIPTION
**Pull request by asiia-trilitech**
## Proposed changes

[Task link](https://app.asana.com/0/1205770721172203/1206403075254839/f)

## Types of changes

- [ ] Bugfix
- [ ] New feature
- [x] Refactor
- [ ] Breaking change
- [ ] UI fix

## Screenshots

Add the screenshots of how the app used to look like and how it looks now

| Icon | Before | Now |
| ---- | ------ | --- |
|  Ledger |   Screenshot 2024-03-11 at 15 22 59      |    Screenshot 2024-03-13 at 12 08 04   |
| Ledger |     Screenshot 2024-03-11 at 16 13 34       |   Screenshot 2024-03-13 at 12 08 14    |
| Ledger | Screenshot 2024-03-13 at 12 17 18 | Screenshot 2024-03-13 at 12 16 29 |
| Google | Screenshot 2024-03-11 at 17 25 26 |  Screenshot 2024-03-13 at 11 18 05 |
|  Google |  Screenshot 2024-03-11 at 17 25 34  |   Screenshot 2024-03-13 at 11 18 13   |
| ThreeDots |    Screenshot 2024-03-11 at 18 18 10     |  Screenshot 2024-03-13 at 11 25 57   |
| ThreeDots |       Screenshot 2024-03-11 at 18 18 16     |    Screenshot 2024-03-13 at 11 26 05  |
| ThreeDots |     Screenshot 2024-03-11 at 18 18 24      |    Screenshot 2024-03-13 at 11 26 28   |
|  OutlineExclamationCircle  |   Screenshot 2024-03-12 at 11 45 20   |   Screenshot 2024-03-13 at 11 45 19    |
|   CopyableAddress  |    Screenshot 2024-03-12 at 10 53 57       |  Screenshot 2024-03-12 at 10 57 27   |
|   CopyableAddress  |       Screenshot 2024-03-12 at 10 54 06     |    Screenshot 2024-03-12 at 10 56 31   |

## Checklist

- [ ] Tests that prove my fix is effective or that my feature works have been added
- [ ] Documentation has been added (if appropriate)
- [ ] Screenshots are added (if any UI changes have been made)
- [ ] All TODOs have a corresponding task created (and the link is attached to it)

### CODE REVIEW

File: src/assets/icons/OutlineExclamationCircle.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Icon, IconProps } from "@chakra-ui/react";
+
+import colors from "../../style/colors";
+
+export const OutlineExclamationCircleIcon: React.FC<IconProps> = props => (
+  <Icon
+    width="16px"
+    height="16px"
+    fill={colors.white}
+    viewBox="0 0 1024 1024"
```
Comment (serjonya-trili) [inline src/assets/icons/OutlineExclamationCircle.tsx:10]: these numbers look quite odd. where'd get them from?

File: src/components/CopyableText.tsx [hunk 2]
```diff
@@ -46,15 +44,10 @@ const CopyableText: React.FC<
...
       id: TOAST_ID,
-      render: () => (
-        <ToastBody
-          message={toastMessage}
-          onClose={() => {
-            toast.close(TOAST_ID);
-          }}
-        />
-      ),
+      isClosable: true,
```
Comment (serjonya-trili) [inline src/components/CopyableText.tsx:50]: 👍

Comment (serjonya-trili) [reply src/components/CopyableText.tsx:50]: though on the screenshots the cross has be centered, and might be also smaller than it has to be
the checkmark is filled with black, but should have transparent background

Comment (asiia-trilitech) [reply src/components/CopyableText.tsx:50]: I checked with Bahador & using the chakra confirmation toast is OK for this case.

Ideally we would like to switch to an address pill here - I've created a ticket for that.
https://app.asana.com/0/1205770721172203/1206772726685952/f

And only have CopyableText on the AddressBook page.

File: src/assets/icons/OutlineExclamationCircle.tsx (hunk 1 above)
Comment (asiia-trilitech) [reply src/assets/icons/OutlineExclamationCircle.tsx:10]: Original react icons have this numbers: https://react-icons.github.io/react-icons/search/#q=AiOutlineExclamationCircle

Screenshot 2024-03-13 at 10 49 07

Screenshot 2024-03-13 at 10 50 13

**Review by serjonya-trili — APPROVED** [review]

**Pull request merged**